<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13368" started-at="2020-06-03T19:15:44 IST" finished-at="2020-06-03T19:15:57 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13368" started-at="2020-06-03T19:15:44 IST" finished-at="2020-06-03T19:15:57 IST">
      <class name="com.bayerapi.testcases.AmrutaPOC">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.bayerapi.testcases.AmrutaPOC@1c3a4799]" name="setup" is-config="true" duration-ms="101" started-at="2020-06-03T19:15:44 IST" finished-at="2020-06-03T19:15:44 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="getAreaCropGrape()[pri:0, instance:com.bayerapi.testcases.AmrutaPOC@1c3a4799]" name="getAreaCropGrape" duration-ms="13180" started-at="2020-06-03T19:15:44 IST" finished-at="2020-06-03T19:15:57 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [REGULAR PLAN - GROWT] but found [REGULAR PLAN - GROWTH]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [REGULAR PLAN - GROWT] but found [REGULAR PLAN - GROWTH]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:563)
at com.bayerapi.testcases.AmrutaPOC.getAreaCropGrape(AmrutaPOC.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAreaCropGrape -->
      </class> <!-- com.bayerapi.testcases.AmrutaPOC -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
